pkg/secrets-store/provider_client.go

```diff
@@ -181,5 +182,16 @@ func MountContent(ctx context.Context, client v1alpha1.CSIDriverProviderClient,
 	for _, v := range ov {
 		objectVersions[v.Id] = v.Version
 	}
+
+	if len(resp.GetFiles()) > 0 {
```

Comment:
if no file is returned, do we bubble up the error? We should at least log it.

Reply:
> if no files are returned the assumption is that it is a plugin that has not migrated to this method yet.

That makes sense. Can you pls add this as a comment as this was not obvious to me. Thanks!

Reply:
Done!